cpp/unreal_plugins/CoreUtils/Source/CoreUtils/Config.cpp

```diff
@@ -21,19 +22,26 @@ void Config::initialize()
     // If a config file is provided via the command-line, then load it
     if (FParse::Value(FCommandLine::Get(), TEXT("config_file="), config_file)) {
         s_config_ = YAML::LoadFile(Unreal::toStdString(config_file));
+        is_enabled_=true;
 
     // Otherwise, if MyProject/Temp/config.yaml exists, then load it
```

Comment:
I think we can get rid of this `else if` statement, and we can also remove `generate_config.py` from our build steps. Nice!

Reply:
Update: we can get rid of this `else if` statement but not `generate_config.py`.